ocs_ci/helpers/helpers.py

```diff
@@ -713,7 +714,24 @@ def create_multiple_pvcs(
     )
     time.sleep(number_of_pvc)
 
-    return ocs_objs
+    return ocs_objs, tmpdir
```

Comment:
pleas update the return argument  in the docstring to reflect this change

Reply:
Done. Thanks